ansible/roles/piwik/templates/config.ini.php.j2

```diff
@@ -0,0 +1,68 @@
+; <?php exit; ?> DO NOT REMOVE THIS LINE
+; file automatically generated or modified by Piwik; you can manually override the default values in global.ini.php by redefining them in this file.
+[database]
+host = "{{ mysql_host }}"
+username = "{{ mysql_piwik_user }}"
+password = "{{ mysql_piwik_pass }}"
+dbname = "{{ mysql_piwik_db }}"
```

Comment:
ansible variables live in global namespace, `mysql_host` sounds like a imperfect name :)